plugins/codemirror/dialogs/codemirrordialog.js

```diff
@@ -199,9 +207,7 @@ CKEDITOR.dialog.add('codemirrordialog', function (editor) {
 			var editor = this.getParentEditor();
 			var data = editor.getData();
 
-			if (codeMirrorEditor && codeMirrorEditor.getValue() !== data) {
```

Comment:
What I understand we're doing here is checking if codeMirror exists and update the source editor content only if its current value is different from the editor one. Why we need to remove this?